tests/e2e/createWallets.spec.ts

```diff
@@ -0,0 +1,198 @@
+import { test } from "@playwright/test";
+import puppeteer from "puppeteer";
+import { getDocument, queries, wait } from "pptr-testing-library";
+const { getByText, getByLabelText } = queries;
+import { USER } from "complete-randomer";
+
+const delay = async (time) => {
+  return new Promise(function (resolve) {
+    setTimeout(resolve, time);
+  });
+};
+
+const loadExtension = async () => {
+  const extensionID = "illmemocmhbdlcejaopkmeegcbjbnmnm";
```

Comment:
Had to modify the ID to make it work.
@bumi do you know when this changes? Depending on that we need to think about how to handle this a somewhat automatic way in the future.

Reply:
I think on chrome development this changes every new install? or at least is different on every computer.
Only the production one is fixed afaik.